packages/playground/website/playwright/e2e/blueprints.spec.ts

```diff
@@ -339,7 +339,6 @@ test('HTTPS requests via curl_exec() should work', async ({
 					curl_setopt($ch, CURLOPT_TCP_NODELAY, 0);
 					curl_setopt($ch, CURLOPT_RETURNTRANSFER, 1);
 					$result = curl_exec($ch);
-					curl_close($ch);
```

Comment:
Since PHP 8.0 `curl_close` isn't required as the handle is released automatically when the CurlHandle object falls out of scope or is garbage-collected. ([source](https://levelup.gitconnected.com/preparing-for-php-8-5-the-curl-close-deprecation-and-how-to-refactor-67ea89d59850))

I'm not sure if I should add `curl_close` for PHP 7 tests, or if it's ok to leave it as-is.